### ISSUE TITLE
Truncated display of count number in Paned Browser

### ISSUE THREAD

**Issue by urielz** [quodlibet/quodlibet#1750]
In Paned Browser view, the count number is not display correctly on some columns. As an example, see attached imaged: the column “Genre” has several categories with 3 digit numbers, but only the first two digits can be seen (highlighted in red)

version: QL 3.5.1 on OS X 10.11.1 (but I should say that I have seen this bug in previous versions of QL as well)

![ql_pbrowser](https://cloud.githubusercontent.com/assets/11670888/11407169/d7ca76ca-9365-11e5-8128-35e9b30a8d8f.jpg)

**Comment by lazka** [comment]
Yeah, that's a GTK+ bug and been there since early gtk+3.x. As I haven't found a way to reliably reproduce this I haven't reported it upstream.

Something related (also about wrong cell widths) I reported is https://bugzilla.gnome.org/show_bug.cgi?id=701282 which might be related

**Comment by urielz** [comment]
Thanks for the info. It doesn't bother me, but perhaps a temporary workaround is to force a blank space character after the closing parenthesis?

**Comment by lazka** [comment]
Sounds good.

**Comment by lazka** [comment]
For future reference: it depends on the length of the entries when first displayed. If the first view only contains short text scrolling to the longer ones shows them truncated.

**Comment by lazka** [comment]
Reproducer: https://gist.github.com/lazka/75483c521457f4a5db1c
